File: Java/634.java
import java.util.Scanner;

/**
 *
 * 634 - Bandurria Hero - Golgotha's Melancholy - https://github.com/Golgotha-sMelancholy
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int casos = sc.nextInt();
        for (int i = 1; i<=casos; i++) {
            String notas = sc.next();
            int combo = 10;
            int resultado = 0;
            for (int j = 0; j<notas.length(); j++) {
                if (notas.charAt(j) == 'O') {
                    resultado += combo;
                    combo += 10;
                } else {
                    combo = 10;
                }
            }
            System.out.println(resultado);
            
        }

    }
}

File: C++/634.cpp
#include <iostream>

using namespace std;

// 634 - Bandurria Hero - Golgotha's Melancholy - https://github.com/Golgotha-sMelancholy

int main()
{
    int loop = 0;
    cin >> loop;
    string chords = "";

    for (int i = 0; i<loop; i++) {
        cin >> chords;
        int bonus = 1;
        int combo = 10;
        int points = 0;
        for (int j = 0; j<chords.length(); j++) {
            if (chords[j] == 'O') {
                points += combo * bonus;
                bonus++;
            } else {
                bonus = 1;
            }
        }
        cout << points << endl;
    }
    return 0;
}
